### DISCUSSION TITLE
Pending label in repository

### DISCUSSION THREAD

**Discussion by TeslaCoil9001** [studio-minus/ppg-bugs-and-requests#2061] [Ideas]
Add a "Pending" label for ideas that have been considered for the game before they're accepted or rejected so we aren't left thinking they're ignored

**Comment by dinevolence** [comment]
Perfect for @PossiblyRemember

**Comment by PossiblyRemember** [comment]
> Perfect for @PossiblyRemember

well, you're not wrong

**Comment by mestiez** [comment]
Nice idea
